src/components/Demo/Feature.tsx

```diff
@@ -0,0 +1,78 @@
+import { demoData } from "../../constants/demoData";
+import "./feature.css";
+import flashcard from "../../assets/demo/flashcard.png";
+import chatbot from "../../assets/demo/chatbot.png";
+import { Tilt } from "react-tilt";
+import summary from "../../assets/demo/summary.png";
+import quiz from "../../assets/demo/quiz.png";
+import screenshot from "../../assets/screenshot.png";
+
+const Feature = ({ id }: { id: number }) => {
+  const data = demoData[id - 1];
+
+  return (
+    <>
+      <div
+        className={` text-left feature-container flex text-white  ${!(id % 2) ? "flex-row light-background " : "flex-row-reverse"} `}
+      >
+        <div className="description-container w-[50%]">
+          <div className={`positioning  ${!(id % 2) ? null : "ml-[5%] "} `}>
+            <div className="icon-container  ml-[30%] mb-[50px] ">
+              <img
+                className="icon w-[35%] h-[35%]"
+                src={
+                  data.topic === "flashcard"
```

Comment:
question: aaleh mat7otch object el taswira direct fi wost el data, w taffichih direct 
twali src={data.icon} w maghyr barcha ifs

Reply:
base hatena khamamt akeka milouel ama jarabtha several times w it  didnt work